webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js

```diff
@@ -781,6 +852,8 @@ export class SakaiGrader extends gradableDataMixin(SakaiElement) {
       this.submitGradingData(formData);
       const rubricGrading = document.getElementsByTagName("sakai-rubric-grading").item(0);
       rubricGrading && rubricGrading.release();
+      document.getElementById("feedback-active-indicator").classList.remove("unsaved");
+      document.getElementById("pvt-notes-active-indicator").classList.remove("unsaved");
```

Comment:
Are you sure these elements are always present?

Reply:
I changed this by two properties in the component, now is updated when the properties are updated (I think it is much better now)